docs/src/components/productX/XGridFullDemo.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { DataGridPro, GridToolbar } from '@mui/x-data-grid-pro';
```

Comment:
Changes in this file are mostly related to broken pagination:

https://github.com/mui/material-ui/assets/13808724/eff8f14f-ed50-4b33-8a34-a3cbcf639996